pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -852,17 +852,11 @@ func (npm *nominatedPodMap) UpdateNominatedPod(oldPod *v1.Pod, newPodInfo *frame
 	npm.add(newPodInfo, nodeName)
 }
 
-// NewPodNominator creates a nominatedPodMap as a backing of framework.PodNominator.
-// DEPRECATED: use NewSafePodNominator() instead.
-func NewPodNominator() framework.PodNominator {
-	return NewSafePodNominator(nil)
-}
-
-// NewSafePodNominator creates a nominatedPodMap as a backing of framework.PodNominator.
-// Unlike NewPodNominator, it passes in a podLister so as to check if the pod is alive
+// NewSafePodNominator creates a nominator as a backing of framework.PodNominator.
+// A podLister is passed in so as to check if the pod is alive
 // before adding its nominatedNode info.
 func NewSafePodNominator(podLister listersv1.PodLister) framework.PodNominator {
```

Comment:
do we really need to continue to use the name "Safe"? we can just use `NewPodNominator` or `NewNominator`.

Reply:
(though of it as well)

In that case, we will have different versions of NewNominator() in master vs. cherry-picked lower versions (1.19 ~ 1.21), but maybe not that concerned?

Reply:
it's not an API.

+1 to removing `Safe` from the name.

Reply:
done.